### DISCUSSION TITLE
FYI: Working well without any problems on 22000.376

### DISCUSSION THREAD

**Discussion by BraINstinct0** [valinet/ExplorerPatcher#571] [General]
Recent Beta Channel update to 376 didn't break anything. Symbol downloads turned on, of course.
However that's totally about 'my setup', so if relevant problems are found feel free to share it here.

**Comment by valinet** [comment]
Actually the current stable releases of Windows 11 is 22000.376. That has been pushed yesterday (Patch Tuesday) as a "B" release to users, so I've got that as well.

https://docs.microsoft.com/en-us/windows/release-health/windows11-release-information

As far as the files requiring symbols, `twinui.pcshell.dll` and `StartDocked.dll`, those are the same as in 22000.348, so they automatically benefited from the built-in offsets, so no symbols had to be downloaded anyway. I bumped up the version for the pre-releases as well, and will promote that as a release soon, if no major problems are found.

Thanks for the report
